recipes/libgit2/all/patches/1.7.2-0001-fix-cmake.patch

```diff
@@ -0,0 +1,70 @@
+diff --git a/cmake/SelectHTTPParser.cmake b/cmake/SelectHTTPParser.cmake
+index 955aea3..ff94eb8 100644
+--- a/cmake/SelectHTTPParser.cmake
++++ b/cmake/SelectHTTPParser.cmake
+@@ -1,10 +1,10 @@
+ # Optional external dependency: http-parser
+ if(USE_HTTP_PARSER STREQUAL "system")
+-	find_package(HTTPParser)
```

Comment:
We could use CMakeDeps with set_property to change the cmake file name, but variables are still not possible to change, so this change looks okay.